scipy/special/tests/test_log_softmax.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+from numpy.testing import (assert_almost_equal, assert_equal, assert_allclose,
+                           assert_array_almost_equal, assert_)
+
+from scipy.special import log_softmax
+
+def test_log_softmax():
```

Comment:
The first two cases could be handled by a `pytest.mark.parametrize`; see below for an example.